driver-legacy/src/main/com/mongodb/MongoClient.java

```diff
@@ -732,10 +732,15 @@ private static Cluster createCluster(final ClusterSettings clusterSettings, @Nul
                 credential,
                 getCommandListener(options.getCommandListeners()),
                 options.getApplicationName(),
-                mongoDriverInformation,
+                wrapMongoDriverInformation(mongoDriverInformation),
                 options.getCompressorList());
     }
 
+    private static MongoDriverInformation wrapMongoDriverInformation(@Nullable final MongoDriverInformation mongoDriverInformation) {
+        return (mongoDriverInformation == null ? MongoDriverInformation.builder() : MongoDriverInformation.builder(mongoDriverInformation))
```

Comment:
This fixes is a regression introduced when I merged the Mongo class into MongoClient

Reply:
Ack - good catch